docs/reference-guide/streams/overview.md

```diff
@@ -140,3 +140,23 @@ the Kafka cluster. However, even though we cannot guarantee **exactly once**
 semantics, we tried to minimize the possibility of processing messages multiple
 times. This means committing the message offsets to the Kafka cluster happens
 right after the transaction is committed to the database.
+
+## Configuring stream transactions
+
+A stream fails when a transformation fails to commit a transaction. This is a side effect
+of [isolation levels](/reference-guide/isolation-levels.md) and its behaviour can be
+configured by setting the appropriate memgraph flag:
```

Comment:
It suggests me that a transaction can only fail because of conflict, but that is not true. A transaction can fail because of various reasons. I would rephrase this as:
```suggestion
A stream might fail when a transaction (in which the returned queries are executed) fails to commit because of another conflicting transaction. This is a side effect
of [isolation levels](/reference-guide/isolation-levels.md) and its behaviour can be
configured by setting the appropriate memgraph flag:
```

It is a bit rough, so feel free to rephrase it.